massa-network/src/binders.rs

```diff
@@ -96,7 +95,7 @@ impl ReadBinder {
     /// or =0 if there is no more data.
     /// We can't use read_exact and similar because they are not cancel-safe:
     /// https://docs.rs/tokio/latest/tokio/io/trait.AsyncReadExt.html#cancel-safety-2
-    pub async fn next(&mut self) -> Result<Option<(u64, Message)>, NetworkError> {
+    pub async fn next(&mut self) -> Result<Option<(u64, Message, Option<Vec<u8>>)>, NetworkError> {
```

Comment:
What's the option you added for ?

Reply:
That is the serialized object, would need some documentation...